front-end/src/components/ProductSelection/ProductCard.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { Text, Icon, Button, View } from 'native-base'
+import PropTypes from 'prop-types'
+import { Alert } from 'react-native'
+
+import { tinyStyle, extendStyle } from './ProductSelectionStyle'
+
+export default function ProductCard(props) {
+  function handleSelectClick(id) {
+    Alert.alert(`${id} is clicked!`)
```

Comment:
@puteulanus instead of alerting navigate back to the new loan's screen (or just navigate back)